docker-compose.yml

```diff
@@ -0,0 +1,6 @@
+web:
```

Comment:
Why is this file in the root of the blueprints repository?

Reply:
The thought of having a docker-compose.yml is to provide some guidance on what type of infrastructure would be required to use this blueprint.  Furthermore, using the docker-compose.yml allows one to have a working example of this blueprint when applied against the docker instances created.

Reply:
I agree there. It's even in the DOD.

But this `docker-compose.yml` file is in the root directory of the repository, not in the directory for the blueprint in a subdirectory called `docker`.

Also, what role does that `web` container play in the blueprints?